test/e2e/search.spec.js

```diff
@@ -21,37 +21,47 @@ test.beforeEach(async ({ context }) => {
 
 // TODO(obulat): Fix the search metadata before search
 //  This test currently passes, although everything should be the opposite
-test('does not show an error message before search', async ({ page }) => {
-  await page.goto('/search')
+// test('does not show an error message before search', async ({ page }) => {
```

Comment:
You can add `.skip` to a test that doesn't pass instead of commenting it out:
`test.skip('shows search result metadata',` ...